modules/claims_api/app/controllers/claims_api/v1/claims_controller.rb

```diff
@@ -40,6 +39,15 @@ def show
 
       private
 
+      def fetch_or_error_local_claim_id
+        claim = ClaimsApi::AutoEstablishedClaim.find_by(id: params[:id])
+        if claim && claim.status == 'errored' && claim.evss_response.any?
+          render json: { errors: format_evss_errors(claim.evss_response) },
+                 status: :unprocessable_entity
+        end
+        claim.try(:evss_id)
+      end
+
```

Comment:
Nothing looks to have changed since `v0`, why not re-route to one version or another instead of duplicating code? Is there a general plan/roadmap for `v1` and can you link to it if so?

Reply:
I added this functionality to Claims v1 as well, just needed to get the tests passing.